client/Components/Form.jsx

```diff
@@ -56,7 +56,7 @@ const ConnectedForm = (props) => {
           Checkout
           <input type="date" onChange={onCheckoutChange} id="checkout" />
         </label>
-        <GuestDropdown duration={duration} data={data} />
+        <GuestDropdown duration={duration} maxGuests={data.maxGuests} />
```

Comment:
you should destructure maxGuests above, surprised eslint didn't throw an error.

Reply:
I'm surprised too, but it didn't. I'll fix that.

Reply:
Actually, it may do that to avoid needing to double-deconstruct, which I'd have to do here.

I'll do that anyway, though, for consistency's sake.